doc/ug/getting_started_dv.md

```diff
@@ -61,28 +60,28 @@ There is support for Cadence Xcelium as well, which is being slowly ramped up.
 
 ## Building and Running Tests
 
-The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<ip>_sanity_vseq.sv` for developing the sanity test.
-The sanity test can be run as-is by invoking `make`, as a "hello world" step to bring the DUT out of reset.
+The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<dut>_sanity_vseq.sv` for developing the sanity test.
+The sanity test can be run as-is by invoking `dvsim.py`, as a "hello world" step to bring the DUT out of reset.
 ```console
 $ cd path-to-dv
-$ make [SIMULATOR=xcelium] [WAVES=1]
+$ util/dvsim.py <dut>_sim_cfg.hjson -i <dut>_sanity [-w] [-t xcelium]
```

Comment:
```suggestion
$ util/dvsim/dvsim.py path/to/<dut>_sim_cfg.hjson -i <dut>_sanity [--waves] [--tool xcelium]
```

Reply:
Good catch, thanks Sri, sorry I missed the dvsim folder.